packages/rlc-codegen/src/interfaces.ts

```diff
@@ -35,3 +35,54 @@ export interface OperationAnnotations {
   isLongRunning?: boolean;
   isPageable?: boolean;
 }
+export interface RLCModel {
+  libraryName: string;
+  srcPath: string;
+  paths: Paths;
+  schemas?: Schema[];
+}
+export interface File {
+  path: string,
+  content: string
+}
+
+export enum SchemaContext {
+  /** Schema is used as an input to an operation. */
+  Input = "input",
+  /** Schema is used as an output from an operation. */
+  Output = "output",
+  /** Schema is used as an exception from an operation. */
+  Exception = "exception"
+}
+
+export interface Schema {
+  name: string;
+  type: string;
+  description?: string;
+  required?: boolean;
+  default?: any;
+  readOnly?: boolean;
+  usage?: SchemaContext[];
+}
+
+export interface ObjectSchema extends Schema {
```

Comment:
@qiaozha Just wondering if this is an ObjectSchema, its name will be the same as type, right? For example the model is `Car` then its name and type will be "Car".